__tests__/app.test.js

```diff
@@ -336,3 +338,40 @@ describe("GET: /api/users", () => {
    });
  });
 });
+describe("GET: /api/articles/:article_id(comment_count)", () => {
+ test("should respond with a 200 status code and article object", () => {
+  return request(app)
+   .get("/api/articles/3")
+   .expect(200)
+   .then(({ body }) => {
+    expect(body.article).toEqual({
+     article_id: 3,
+     title: "Eight pug gifs that remind me of mitch",
+     topic: "mitch",
+     author: "icellusedkars",
+     body: "some gifs",
+     created_at: "2020-11-03T09:12:00.000Z",
+     votes: 0,
+     article_img_url:
+      "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+     comment_count: "2",
+    });
+   });
```

Comment:
again try to use objectContaining or something less brittle as a matcher here instead of toEqual